programs/fileio.c

```diff
@@ -566,6 +566,7 @@ static FILE* FIO_openDstFile(FIO_prefs_t* const prefs, const char* srcFileName,
     {   FILE* const f = fopen( dstFileName, "wb" );
         if (f == NULL)
             DISPLAYLEVEL(1, "zstd: %s: %s\n", dstFileName, strerror(errno));
+        chmod(dstFileName, 00600);
```

Comment:
I understand that `chmod()` can fail silently,
yet it would feel cleaner if `chmod()` wasn't attempted when `f == NULL`.
an `if { } else { } ` should do the trick.

Reply:
Totally right, I did miss that. :)